ui/components/component-library/text/text.tsx

```diff
@@ -42,7 +42,6 @@ export const Text: TextComponent = React.forwardRef(
       ellipsis,
       className = '',
       children,
-      isBrandEvolution, // Enables Brand Evolution Typography do not use unless you are working on the brand evolution
```

Comment:
Removing temporary feature flag that enables the brand evolution now that everything will use the brand evolution font family `CentraNo1`